packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -529,13 +535,14 @@ export function unpackRuntimeMessage(message: ISequencedDocumentMessage) {
             message.type = innerContents.type;
             message.contents = innerContents.contents;
         }
-        assert(isUnpackedRuntimeMessage(message), 0x122 /* "Message to unpack is not proper runtime message" */);
+        return true;
```

Comment:
#nit this function is kind of weird because it mutates the parameter as a side effect and then returns true/false if the message is an op. Maybe the boolean return can be a separate function or just a comparison of the `message.type`.

Reply:
The verb in name "unpack" tells what it does, so I think it's fine from that POV. We already have that other function - isRuntimeMessage(), if you feel strong about it, I can change it, but all the places that call this API would need to call both functions, that seems like a waste to me

Reply:
I think at least adding function docs that make it clear that the parameter is mutated in-place (bonus points if it describes *how* it changes), is worth it.

Reply:
And now that I think about it, also what the return value means.

The alternative of returning the unpacked message and keeping the parameter untouched does feel more friendly to new programmers (less bug prone) but we might not want to have to allocate to clone the message. I think FF in general is complex enough that I don't mind going with the current approach.